pvlib/tests/ivtools/test_sdm.py

```diff
@@ -379,4 +380,20 @@ def test__calc_theta_phi_exact_vector():
     assert np.isnan(theta[0])
     assert np.isnan(theta[1])
     assert np.isnan(phi[0])
-    np.testing.assert_allclose(phi[1], 2.2079, atol=.0001)
+    assert_allclose(phi[1], 2.2079, atol=.0001)
+
+
+def test_pvsyst_temperature_coeff():
+    # test for consistency with dP/dT estimated with secant rule
+    params = {'alpha_sc': 0., 'gamma_ref': 1.1, 'mu_gamma': 0.,
+              'I_L_ref': 6., 'I_o_ref': 5.e-9, 'R_sh_ref': 200.,
+              'R_sh_0': 2000., 'R_s': 0.5, 'cells_in_series': 60}
+    params_lowT = pvsystem.calcparams_pvsyst(1000, 24, **params)
+    params_hiT = pvsystem.calcparams_pvsyst(1000, 26, **params)
+    res_lowT = pvsystem.singlediode(*params_lowT)
+    res_hiT = pvsystem.singlediode(*params_hiT)
+    expected = (res_hiT['p_mp'] - res_lowT['p_mp']) / 2.
+    # convert to %/C
+    expected = expected * 100 / (0.5 * (res_hiT['p_mp'] + res_lowT['p_mp']))
```

Comment:
I'd prefer for this value to be hard coded and for the function calls above to be removed.